components/server/src/main/java/com/hotels/styx/server/AdminHttpRouter.java

```diff
@@ -0,0 +1,54 @@
+/*
+  Copyright (C) 2013-2019 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.server;
+
+import com.hotels.styx.api.Eventual;
+import com.hotels.styx.api.HttpHandler;
+import com.hotels.styx.api.HttpInterceptor;
+import com.hotels.styx.api.LiveHttpRequest;
+import com.hotels.styx.api.LiveHttpResponse;
+import com.hotels.styx.api.WebServiceHandler;
+import com.hotels.styx.common.http.handler.HttpAggregator;
+
+import static com.hotels.styx.api.HttpResponse.response;
+import static com.hotels.styx.api.HttpResponseStatus.NOT_FOUND;
+
+/**
+ * Simple Http Router.
+ */
+public class AdminHttpRouter implements HttpHandler {
```

Comment:
Hi @taer Thank you for doing this! Very useful. But one comment. 

Please move this class to `styx-proxy` module. This class is called *Admin*HttpRouter, and you are adding it to the `styx-server` module which doesn't have an admin interface. The admin interface is in the `styx-proxy`. Therefore it should live in the proxy module. But otherwise it good to go :-D.

Reply:
done. I think the test failure is a timeout based test